lib/rubocop/ast/node/send_node.rb

```diff
@@ -10,7 +10,8 @@ class SendNode < Node
       include MethodDispatchNode
 
       def_node_matcher :attribute_accessor?, <<~PATTERN
-        (send nil? ${:attr_reader :attr_writer :attr_accessor :attr} $...)
+        [(send nil? ${:attr_reader :attr_writer :attr_accessor :attr} $...)
+         (_    _    _                                                 _ ...)]
```

Comment:
Should we have an issue that collects all the missing constructs in NodePattern?

In this case seems like we're missing the ability to match "one or more of the following".

Reply:
Yes, good point. I wonder how often it actually happens. I was thinking that performance-wise it was basically not an issue, but clearly this is kind of obscure.

I was also wondering how useful it was to return `[:attr, an_array_of_nodes]` instead of `[:attr, *an_array_of_nodes]`. I'm still unclear as to how much compatibility we want to keep between releases...